### PR TITLE
Update CODEOWNERS

### PR DESCRIPTION
**Pull request by ruowan**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -422,7 +422,7 @@
 # ServiceOwners:                                                   @a-t-mason @ganzee @manuaery
 
 # ServiceLabel: %Data Factory
-# ServiceOwners:                                                   @shawnxzq @lmy269
+# ServiceOwners:                                                   @ruowan @Frey-Wang @shawnxzq
```
Comment (jsquire) [inline .github/CODEOWNERS:425]: - The account `ruowan` needs to have the Azure organization made public on their profile.
- The account `Frey-Wang` needs to have the Microsoft and Azure organizations made public on their profile.

Both accounts are confirmed to have "write" permissions already.

**Review by jsquire — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (ruowan) [reply .github/CODEOWNERS:425]: The two account profile is updated.

Comment (jsquire) [reply .github/CODEOWNERS:425]: `Frey-Wang` has made their `Azure` membership public.   They still need to make their `Microsoft` membership public.

**Pull request closed without merging**